app/src/main/res/layout/tool_detail_banner_view.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This appears to be exactly the same as the one in `layout-v17`. Is one of the supposed to not include the video player? Or be different in some way?